packages/ccd-js-gen/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "extends": "../../tsconfig-base.json",
-    "include": ["src/**/*", "package.json"],
+    "include": ["src/**/*", "test/**/*", "package.json", "jest.config.ts"],
     "compilerOptions": {
         "module": "NodeNext",
         "moduleResolution": "NodeNext",
```

Comment:
Any reason why we specify "NodeNext" here? In the base config we have "Node16", and I feel like the better option is to "lock the version" to ensure that our code remains compatible with node 16 and above. Right now, there is no difference in the modules produced or the module resolution strategy applied, but that might change.

Reply:
The reason is subtle, but it stems from importing the `package.json`:
```ts
import packageJson from '../package.json' assert { type: 'json' };
```
For safety reasons, newer versions of NodeJS require adding `assert` when importing anything other than JS.
So to be able to support these newer versions, we need the assert (or workaround the need to import package json).
Version 16 does not support import-assert, so we cannot use `Node16`.